Kajabi/sage-lib
# Sage font updates

## Description
This PR updates the Sage icon font with `add-image` and `folder-group`.

### Screenshots
<img width="143" alt="icon-add-image" src="https://user-images.githubusercontent.com/7331038/102551435-786e6800-408d-11eb-8597-183efd3103c2.png">
<img width="179" alt="icon-folder-group" src="https://user-images.githubusercontent.com/7331038/102551436-7906fe80-408d-11eb-9996-109753ecc9d1.png">

## Test notes
1. Visit http://0.0.0.0:4000/pages/design/icon
2. Ensure that the new icons exist on the page
